cli/ballerina-cli/src/main/resources/cli-help/ballerina-help.help

```diff
@@ -8,31 +8,31 @@ SYNOPSIS
 
 
 DESCRIPTION
-       Ballerina is a statically typed, concurrent programming language, focusing on network interaction and structured
-       data. It is intended to be the core of a language-centric middleware platform. It has all the general-purpose
-       functionality expected of a modern programming language, but it also has several unusual aspects that make it
+       Ballerina is a statically-typed, concurrent programming language focusing on network interaction and structured
+       data. It is intended to be the core of a language-centric middleware platform. It not only has all the general-purpose
+       functionality expected from a modern programming language, but also has several unusual aspects that make it
        particularly suitable for its intended purpose.
 
-       Find more information at: https://ballerina.io/
+       Find more information at: https://ballerina.io/.
```

Comment:
```suggestion
       Find more information at: https://ballerina.io/
```

Reply:
@pubudu91 - What is the reason to not have the slash and the full stop here?

Reply:
Slash is not required. Plus a full stop there would mean it's part of the URL.